docs/topics/multiplatform/multiplatform-expect-actual.md

```diff
@@ -399,6 +399,24 @@ typically provides an expected `CommonViewModel` class whose actual Android coun
 from the Android framework. See [Use platform-specific APIs](multiplatform-connect-to-apis.md#adapting-to-an-existing-hierarchy-using-expected-actual-classes)
 for a detailed description of this example. -->
 
+#### Limitations of open expected classes
```

Comment:
Please drop "open". The rules that you describe are applicable to both open and non-open classes.